test/terraform/gke_test.go.nolint

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Google LLC All Rights Reserved.
```

Comment:
Remind me why we do this sometimes again?

Reply:
The reason behind this is that `go mod tidy` on the root path would change dependencies badly.